src/sentry/utils/suspect_resolutions/analytics.py

```diff
@@ -0,0 +1,23 @@
+from sentry import analytics
+
+
+class SuspectResolutionEvaluation(analytics.Event):
+    type = "suspect_resolution.ids"
+
+    attributes = (
+        analytics.Attribute("resolved_group_id"),
+        analytics.Attribute("suspect_resolution_id"),
+    )
+
+
+class CandidateGroupEvaluation(analytics.Event):
+    type = "candidate_group.ids"
+
+    attributes = (
+        analytics.Attribute("resolved_group_id"),
+        analytics.Attribute("candidate_group_id"),
+    )
```

Comment:
Would it be useful for analysis to include:
* the pearson r value 
* the true/false result of the commit correlation? 
* whether the resolved issue was resolved via UI/commit/release?
* the release id that was deployed that we think resolved the issue
* start and end time for the metric correlation range
 
That way it'll be easier when you do the manual backtesting to verify things are working as expected.

Reply:
Would the release id be a relevant field to record? For commit correlation, we use all the releases that the issue was seen in (or all the releases across a specific time-window for issues that are resolved manually).

Reply:
Yeah, all of the above are suggestions to potentially make it easier for debugging and verifying. Since the extra data isn't particularly expensive to include, maybe release_id might be useful if we're not 100% sure of the association between an issue and a release, i.e. overlapping releases within a time-window. Just something to think about.